packages/elements/src/components/ino-dialog/ino-dialog.tsx

```diff
@@ -87,7 +93,7 @@ export class Dialog implements ComponentInterface {
 
   @Listen('keyup', { target: 'body' })
   handleKeyUp(event: KeyboardEvent) {
-    if (event.key === 'Escape' && this.open && this.dismissible) {
+    if (event.key === 'Escape' && this.open) {
```

Comment:
Why did you change this condition? Should a dialog always be closeable by pressing escape? 🤔

Reply:
Yes, based on the findings of #1006 the `ESC` key should always be able to close the dialog - this is to ensure that keyboard users can easily close the dialog, especially for those who rely on screen readers or cannot use a mouse.